### ISSUE TITLE
Having an issue where the song title and artist name will disappear for a bit before coming back

### ISSUE THREAD

**Issue by abcdeed7** [RobertFrydenlund/SpotifyPlugin#19]
I'm trying to write a simple skin to pull the artist name and song title for my rainmeter setup, but every so often the song title and artist will disappear for a bit before coming back. is this a fixable issue? I'm fairly new to rainmeter and plugins so it's likely I'm missing something simple

Heres the skin I threw together for pulling the artist, the song skin is almost identical save pulling the different data:

`[MeasureArtist]
Measure=Plugin
Plugin=SpotifyPlugin
Type=ArtistName

[Artist]
Meter=String
X=144
MeasureName=MeasureArtist
MeterStyle=styleMetadata
StringStyle=Bold
FontColor=255,255,255,255
FontSize=25
FontFace=Ubuntu
AntiAlias=1
DynamicVariables=1
`

**Comment by RobertFrydenlund** [comment]
Unlucky timing I'm afraid. We are going through some changes right now, so there is not a stable release out at the moment. More info here #18. Will leave this issue open in case it doesn't solve itself when we fix the rest.

**Comment by GibStorm** [comment]
I am seeing the same thing. Actually, it seems to be showing placeholder title/artist data and blank cover image more often than not.

**Comment by RobertFrydenlund** [comment]
Yeah, we are hitting the WebApi ratelimit. The plugin requests updated information twice a second, which I thought was reasonably slow, but apparently not. There is a limit to how slow I can let it be, as it will start to feel laggy when switching songs. Not really sure what to do at the moment...

**Comment by GibStorm** [comment]
So you just gave up? :(

**Comment by RobertFrydenlund** [comment]
See https://github.com/RobertFrydenlund/SpotifyPlugin/issues/18#issuecomment-407540174.
Seems like the only way besides reading memory or modifying the Spotify client itself is running some extra js inside the client. Turned out someone already did that :)

**Comment by CrazyGuyofDoom** [comment]
Can confirm I have the same problem here.
